src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+import java.util.Random;
+
+public class Lottery {
+
+  Random random = new Random();
+  ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
The 'colorSupplier' instance variable should be private to follow best practices regarding encapsulation. Exposing it as package-private (default access) is not recommended.